src/main/resources/com/google/api/codegen/js/sample.snip

```diff
@@ -10,6 +10,10 @@
          https://console.developers.google.com/apis/api/{@class.apiName}
       2. Get access keys for your application. See
          https://developers.google.com/api-client-library/javascript/start/start-js#get-access-keys-for-your-application
+      @if class.auth.instructionsUrl
+        3. For additional information on making an authenticated call, see
```

Comment:
What do you think about s/making an authenticated call/authentication/? To me, the phrasing now slightly suggests that an authenticated call is an optional alternative to a more conventional sort of call.